disnake/types/channel.py

```diff
@@ -53,6 +53,7 @@ class _BaseGuildChannel(_BaseChannel):
     permission_overwrites: List[PermissionOverwrite]
     nsfw: bool
     parent_id: Optional[Snowflake]
+    flags: int
```

Comment:
optional according to docs, though pretty sure I haven't seen that so far

Reply:
Its treated as optional everywhere, might as well change this to mark it as optional.

Reply:
https://github.com/DisnakeDev/disnake/pull/547/commits/e8e04ad9074981e00f332ecd131afd9cca008449